src/utils/send-it.js

```diff
@@ -12,29 +32,44 @@ const deduplicate = (localRawFileNames, existingRawFileNames) => {
   return uniqueRawFileNames;
 };
 
+const uploadUniqueFilesToS3 = async (bucketFolder, localRawFileNames) => {
+  const existingRawFileNames = await s3.listRawFileNames(bucketFolder);
+  const uniqueLocalRawFileNames = deduplicate(localRawFileNames, existingRawFileNames);
+  s3.uploadFolderContents(uniqueLocalRawFileNames, uploadsFolder, targetFolder);
+};
+
 const main = async () => {
   const localRawFileNames = fs.readdirSync(uploadsFolder);
   console.log(`Total number of files found from local folder [${uploadsFolder}]: ${localRawFileNames.length}`);
 
-  // TODO - Simply grabbing both the live and test folders for now.
-  // There is a WIP feature that will change this to be driven by user input.
-  const existingRawFileNames = await s3.listRawFileNames([
-    'staging/',
-    'tweeted/',
-    'quarantine/',
-    'test-staging/',
-    'test-tweeted/',
-    'test-quarantine/',
-  ]);
-  const uniqueLocalRawFileNames = deduplicate(localRawFileNames, existingRawFileNames);
-
-  await s3.uploadFolderContents(uniqueLocalRawFileNames, uploadsFolder, targetFolder);
+  if (!targetFolder) {
+    await inquirer
+      .prompt(questions)
+      .then((userAnswer) => {
+        targetFolder = userAnswer.folderDestination;
+        if (targetFolder === 'test-staging/') {
+          uploadUniqueFilesToS3(testBucketFolders, localRawFileNames);
+        } else if (targetFolder === 'staging/') {
+          uploadUniqueFilesToS3(liveBucketFolders, localRawFileNames);
+        }
+      })
+      .catch((error) => {
+        if (error.isTtyError) {
+          console.log('Prompt could not be rendered in the current environment');
+          console.log(error);
+        } else {
+          console.log(error);
+        }
+      });
+  } else if (targetFolder === 'test-staging/') {
+    uploadUniqueFilesToS3(testBucketFolders, localRawFileNames);
+  } else if (targetFolder === 'staging/') {
+    uploadUniqueFilesToS3(liveBucketFolders, localRawFileNames);
+  } else {
+    console.log(`\nFolder:'${targetFolder}' does not exist`);
+  }
 };
 
 main()
   .then(() => console.log(`Done uploading files to: ${targetFolder}`))
   .catch((e) => console.error(`Error while uploading files to: ${targetFolder} `, e));
```

Comment:
Now that we changed our usage of `targetFolder`, we can no longer easily use the variable here. Let's make this more generic:
```javascript
main()
  .then(() => console.log('Done uploading files!'))
  .catch((e) => console.error('Error while uploading files: ', e));
```

If you'd like a log saying where the files are uploaded to, you can add one at the end of the `uploadUniqueFilesToS3` function.